snaps_openstack/ansible_p/ansible_utils/ansible_configuration.py

```diff
@@ -361,34 +340,26 @@ def launch_provisioning_kolla(iplist, git_branch, kolla_tag, kolla_ansible_tag,
                         'target': controller_ip,
                         'VARIABLE_FILE': variable_file,
                         'BASE_FILE_PATH': consts.KOLLA_SOURCE_PATH})
-            if git_branch.lower() == 'stable/newton':
+            if git_branch.lower() == 'stable/queens':
+                for key,value in host_sriov_interface_node_map.iteritems():
+                  if key is node_ip:
+                     sriov_interface=value
+                     nova_str = ""
+                     sriov_str = ""
+                    
+                     if sriov_interface is not None: 
+                       for iface in sriov_interface:
+                         nova_str = nova_str +  "{\"devname\":\"" + iface +"\", \"physical_network\": \"physnet1\"},"
```

Comment:
Lets try and start using "{}".format(value) method for concatenating strings which is currently the recommended way Python suggests